### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by dderevjanik** [cjshmyr/renegade-2d#1]
Hi, 
The mod seems to be very promising and I like that AI works.
I already played older version of this mod, but currently i'm not able to play it anymore because of absent linux support. 
Is it possible to make linux version ? 

Not sure how build pipeline works, but for example, it could download openra repo and build it with renegade-2 mod resources.

**Comment by cjshmyr** [comment]
Hello!

I've added a bash script (build-maps.sh) that is intended for Linux. Unfortunately I only got to test it against Windows, so wondering if you or someone who runs Linux/OSX could give it a try?

Without passing any arguments it builds the maps in a `bin` folder. If you provide the `--deploy` argument, it attempts to built the map in the appropriate OpenRA maps directory (`home/user/.openra/maps/cnc/playtest-20200329/...`).

The maps in this repository have been updated for the latest [OpenRA playtest](https://www.openra.net/news/playtest-20200329/). The maps on the resource center are a year old, but there will be new versions once the next OpenRA release comes out.

Since the build script is pretty basic (it is just copying files and folders to new locations, and appending text to map.yaml) it could be set up in a pipeline too.
